homeassistant/components/zha/core/channels/lighting.py

```diff
@@ -77,6 +88,21 @@ def current_y(self) -> int | None:
         """Return cached value of the current_y attribute."""
         return self.cluster.get("current_y")
 
+    @property
+    def current_hue(self) -> int | None:
+        """Return cached value of the current_hue attribute."""
+        return self.cluster.get("current_hue")
+
+    @property
+    def enhanced_current_hue(self) -> int | None:
+        """Return cached value of the enhanced_current_hue attribute."""
+        return self.cluster.get("enhanced_current_hue")
```

Comment:
Should this attribute be added to the reporting config?

Reply:
Probably for consistency but we don’t handle color reports yet

Reply:
done